frontend/components/header.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from "react";
+import Image from "next/image";
+
+const PageHeader = ({ imageLoading, imgUrl, title }) => {
+  const [imageLoaded, setImageLoaded] = useState(false);
+
+  useEffect(() => {
+    if (typeof imageLoading === "function") {
+      imageLoading(false);
+    }
+  }, [imageLoaded, imageLoading]);
+
+  return (
+    <div className="coverPhoto">
+      {/* dark overlay */}
```

Comment:
Fix header overlay in smaller screen sizes. No elements should be present. 

![image](https://user-images.githubusercontent.com/80564580/174697415-404ac667-a5d1-4195-948e-5d58eff0f6aa.png)